README.md

```diff
@@ -20,18 +20,26 @@ $ sudo ./install.sh
 ```
 
 ### Installed Tools
----
-* cyberway.eosio-cpp
-* cyberway.eosio-cc
-* cyberway.eosio-ld
-* cyberway.eosio-init
-* cyberway.eosio-abidiff
-* cyberway.eosio-wasm2wast
-* cyberway.eosio-wast2wasm
-* cyberway.eosio-ranlib
-* cyberway.eosio-ar
-* cyberway.eosio-objdump
-* cyberway.eosio-readelf
+
+ * cyberway-abidiff
+ * cyberway-cpp
+ * eosio-abidiff
```

Comment:
There are no eosio-abidiff installed. Only cyberway-abidiff. Same with -cpp.

Reply:
I've just done one more fix for removing those incorrect names/ Please, take a look again